OpenRA.Game/Actor.cs

```diff
@@ -528,7 +528,7 @@ public bool IsTargetableBy(Actor byActor)
 		{
 			// PERF: Avoid LINQ.
 			foreach (var targetable in Targetables)
-				if (targetable.IsTraitEnabled() && targetable.TargetableBy(this, byActor))
+				if (targetable.TargetableBy(this, byActor))
```

Comment:
Wont this change things for many other things?

Reply:
`Targetable.TargetableBy` returns `false` anyway if the trait is disabled, so this should be fine.